### ISSUE TITLE
managing instances of benchmark suites

### ISSUE THREAD

**Issue by nikohansen** [numbbo/numbbo#162]
Currently it seems that we need to implement a new benchmark suite for each instance (i.e. a set of instances of each problem/function) of the GECCO-BBOB testbed. The instance must be reflected in the `SUITE_NAME`, as the problems are extracted like

```coco_problem_t *problem = coco_suite_get_problem(SUITE_NAME, problem_index);```

It seems preferable however to be able to have an instance number in the interface, e.g.

```coco_problem_t *problem = coco_suite_get_problem(SUITE_NAME, suite_instance, problem_index);```

If `suite_instance` is a string, it can "encode" a list of instance numbers which is what we currently do for GECCO-BBOB. We still should implement the testbeds for each workshop with their own `SUITE_NAME` (and empty instance string), but when the above interface is implemented this becomes trivial.

**Comment by brockho** [comment]
seems to make sense to me

**Comment by nikohansen** [comment]
It turns out that we should probably have something like
```
coco_suite_t *suite = coco_suite(SUITE_NAME, suite_instance)
coco_problem_t *problem = coco_suite_get_problem(suite, problem_index)
```

**Comment by brockho** [comment]
blocking #92, #93, #94

**Comment by ttusar** [comment]
Closed by commit a3bc14288a5adfa5dbd3b4ea76a01ce5f9a3c41 to the development branch.
